contrib/k8s/charts/open-service-broker-azure/values.yaml

```diff
@@ -5,7 +5,7 @@ image:
   ## Image location, NOT including the tag
   repository: microsoft/azure-service-broker
   ## Image tag
-  tag: v0.0.1 # This gets automatically overridden by our release process
+  tag: v1.1.0 # This gets automatically overridden by our release process
```

Comment:
@PatrickLang can you revert this? You don't need to update this because the file gets overridden when the chart is released.